tests/base/auto_derivative_function.output

```diff
@@ -22,12 +22,12 @@ f       grad         grads[0]       vgrad[1]      vgrads[1]
 4  1.96e-06 16.03 1.96e-06 16.03 1.96e-06 16.03 1.96e-06 16.03 
 8  1.23e-07 16.01 1.23e-07 16.01 1.23e-07 16.01 1.23e-07 16.01 
 16 7.66e-09 16.00 7.66e-09 16.00 7.66e-09 16.00 7.66e-09 16.00 
```

Comment:
It appears to me that you only replaced those numbers smaller than `1e-10`. Do you want to check your threshold?